storage-resize-images/functions/src/index.ts

```diff
@@ -40,6 +39,18 @@ sharp.cache(false);
 admin.initializeApp();
 
 logs.init();
+
+/**
+ * Supported file types
+ */
+const supportedContentTypes = [
+  "image/gif",
+  "image/jpeg",
```

Comment:
If the image extension is `jpg`, does it still come through as the `jpeg` mime type?

Reply:
Yeah, it is the same content-type for both: https://stackoverflow.com/a/37266399